packages/composite-checkout/src/components/order-review-line-items.js

```diff
@@ -64,10 +64,13 @@ const OrderReviewSectionArea = styled.div`
 `;
 
 function LineItem( { item, className } ) {
+	const itemSpanId = 'checkout-line-item-'.item.id;
```

Comment:
You probably meant to use the `+` operator rather than the dot operator to join strings. Another way to do it which I often find more readable is a template string:

```js
const itemSpanId = `checkout-line-item-${item.id}`;
```

Reply:
Why yes, yes I did. :)